qml/Silica4v-play/qmldir

```diff
@@ -10,7 +10,6 @@ Label 1.0 Label.qml
 ListItem 1.0 ListItem.qml
 MenuItem 1.0 MenuItem.qml
 OpacityRampEffect 1.0 OpacityRampEffect.qml
-Page 1.0 Page.qml
```

Comment:
With every component we move from the Silica wrapper to our own component abstraction, I wonder whether we actually want to stick with this hybrid approach or whether we maybe should move everything from ```Silica4v-play``` to ```components```. But I am not sure. Lets stick with the hybrid approach for now (as it reduces the amount of empty components just sitting there to wrap some different names) and see how we proceed. K?